index.html

```diff
@@ -20,7 +20,6 @@
 
            <h2 class="glad-txt">I'm Frederick Glad to see you!</h2>
```

Comment:
Kindly, this should be your main title, I recommend you use the H1 tag, it is better for the accessibility of your site